README.md

```diff
@@ -4,3 +4,10 @@
 [![arXiv](https://img.shields.io/badge/arXiv-2009.14071-brightgreen)](https://arxiv.org/abs/2009.14071)
 
 The documentation is hosted at [https://zwickytransientfacility.github.io/scope-docs/](https://zwickytransientfacility.github.io/scope-docs/). To generate HTML files of the documentation locally, run `./scope.py doc`
+
+## Funding
+ We gratefully acknowledge previous and current support from the U.S. National Science Foundation (NSF) Harnessing the Data Revolution (HDR) Institute for <a href="https://a3d3.ai">Accelerated AI Algorithms for Data-Driven Discovery (A3D3)</a> under Cooperative Agreement No. <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2117997">PHY-2117997</a>.
+
+ <p align="center">
+ <img src="https://github.com/ZwickyTransientFacility/scope/assets/a3d3.png" alt="A3D3" width="130"/>
```

Comment:
you will need a different path once it is merged I think. but should be easy to fix.

Reply:
Right, will probably have to add something like blob/main to the links